pkg/sensors/tracing/policyhandler_windows.go

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package tracing
+
+import (
+	"errors"
+
+	"github.com/cilium/tetragon/pkg/policyfilter"
+	"github.com/cilium/tetragon/pkg/sensors"
+	"github.com/cilium/tetragon/pkg/tracingpolicy"
+)
+
+type policyHandler struct{}
+
+func init() {
+	sensors.RegisterPolicyHandlerAtInit("tracing", policyHandler{})
+}
+
+func (h policyHandler) PolicyHandler(
+	policy tracingpolicy.TracingPolicy,
+	policyID policyfilter.PolicyID,
+) (sensors.SensorIface, error) {
+
+	policyName := policy.TpName()
+	spec := policy.TpSpec()
+
+	namespace := ""
+	if tpn, ok := policy.(tracingpolicy.TracingPolicyNamespaced); ok {
+		namespace = tpn.TpNamespace()
+	}
+
+	sections := 0
+	if len(spec.KProbes) > 0 {
+		sections++
+	}
+	if len(spec.Tracepoints) > 0 {
+		sections++
+	}
+	if len(spec.LsmHooks) > 0 {
+		sections++
+	}
```

Comment:
The original policyhandler.go code has recently changed. See, for example, https://github.com/cilium/tetragon/blob/main/pkg/sensors/tracing/policyhandler_linux.go#L113-L115.

Were these changes left out intentionally?

Generally, we should avoid duplicating code if we can, so maybe using a helper function would help here.